src/styles/_reset.scss

```diff
@@ -0,0 +1,21 @@
+@import "variables";
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  background-color: $colorBtnFont;
```

Comment:
The variable name `$colorBtnFont` might not be descriptive enough for the purpose of this line. Consider using a more descriptive variable name such as `$backgroundColor`.